infrastructure/ArmTemplates/DeployAll.parameters.json

```diff
@@ -10,37 +10,28 @@
                 "secretName": "AppServiceDiagnosticsLogging"
             }
         },
-        "CompilerHost1":{
+        "CompilerHostEUAP":{
             "reference": {
               "keyVault": {
                 "id": "/subscriptions/0e7b3183-4428-4c1f-93cd-cb2a26616a8a/resourcegroups/AppServiceDiagnosticsResources/providers/Microsoft.KeyVault/vaults/DevOpsDeploymentKV"
               },
               "secretName": "DiagCompilerhostNameEUAP"
             }
         },
-        "CompilerHost2":{
-            "reference": {
-              "keyVault": {
-                "id": "/subscriptions/0e7b3183-4428-4c1f-93cd-cb2a26616a8a/resourcegroups/AppServiceDiagnosticsResources/providers/Microsoft.KeyVault/vaults/DevOpsDeploymentKV"
-              },
-              "secretName": "DiagCompilerhostNameWUS"
-            }
-        },
-        "RuntimeHost1":{
+        "RuntimeHostEUAP":{
             "reference": {
               "keyVault": {
                 "id": "/subscriptions/0e7b3183-4428-4c1f-93cd-cb2a26616a8a/resourcegroups/AppServiceDiagnosticsResources/providers/Microsoft.KeyVault/vaults/DevOpsDeploymentKV"
               },
               "secretName": "DiagRuntimehostNameEUAP"
             }
         },
-        "RuntimeHost2":{
-            "reference": {
-              "keyVault": {
-                "id": "/subscriptions/0e7b3183-4428-4c1f-93cd-cb2a26616a8a/resourcegroups/AppServiceDiagnosticsResources/providers/Microsoft.KeyVault/vaults/DevOpsDeploymentKV"
-              },
-              "secretName": "DiagRuntimehostNameWUS"
-            }
-        }
+        "RuntimeHostProd1": "todo1",
```

Comment:
These names need to be in a key vault so we don't expose ourselves to a denial of service attack. But I don't have access to the key vault